pkg/transformers/json/transformer_test.go

```diff
@@ -22,7 +22,7 @@ const (
 
 func TestTransformJSON(t *testing.T) {
 	now := time.Now().Unix()
-	tr := json.New()
+	tr := json.New(true)
```

Comment:
Test cases for JSON Transformer without flattening are missing. Other than that, this looks good.